src/logic/treasury-and-delegation/use-treasury-and-delegation.ts

```diff
@@ -32,12 +32,16 @@ export const useTreasuryAndDelegation = () => {
         });
       }
 
+      const delegate = isZeroAddress(data.delegate) ? null : data.delegate;
+      const goDelegateName = delegate ? await go(provider.lookupAddress(delegate)) : null;
```

Comment:
Kinda weird seeing the await the promise or just returning null as a ternary 😄

Reply:
It is what it is :D